controllers/capabilities/tenant_internal_reconciler.go

```diff
@@ -90,6 +137,14 @@ func (r *TenantInternalReconciler) reconcileTenant() (*porta_client_pkg.Tenant,
 	return tenantDef, nil
 }
 
+func (r *TenantInternalReconciler) confirmTenantDeleted(tenant *porta_client_pkg.Tenant) (bool, error) {
+	if tenant.Signup.Account.State == "scheduled_for_deletion" {
+		return true, nil
+	} else {
```

Comment:
The only complaint from a [CodeClimate](https://codeclimate.com/github/3scale/3scale-operator/pull/715) that is related to this change is this `else` statement not being on a new line, but I don't think it matters that much :thinking: